tests/testthat/test-Model-class.R

```diff
@@ -355,11 +330,11 @@ test_that("MCMC computes correct values for LogisticNormalMixture model", {
 })
 
 test_that("MCMC computes correct values for LogisticNormalMixture model and empty data", {
-  empty_data <- h_get_data_mixture(empty = TRUE)
+  data <- h_get_data_mixture(empty = TRUE)
```

Comment:
I added a lot of new "empty data" tests with this PR and thought `data` would be better, so in order to keep the consistency I also changed existing ones.

Reply:
yeah that is a bit too much I think. if you really want to do that, make it in a separate issue and PR. otherwise I don't find the relevant changes anymore in your PRs :-)

Reply:
Sure, I will do that. Tx!